dev-proxy/ConsoleLogger.cs

```diff
@@ -65,15 +66,15 @@ public void LogError(string message)
         Console.ForegroundColor = _color;
     }
 
-    public void LogDebug(string message)
+    private void LogDebug(string message)
     {
         if (LogLevel > LogLevel.Debug)
         {
             return;
         }
 
         Console.ForegroundColor = ConsoleColor.Gray;
-        Console.Error.WriteLine(message);
+        Console.WriteLine(message);
```

Comment:
Writing debug message to stderr was on purpose, because they're not considered a part of the command output. Let's reverse this change.

Reply:
Changed back